uploadfs.js

```diff
@@ -65,16 +64,45 @@ function Uploadfs() {
       self.cdn = options.cdn;
     }
 
+
     // Load image backend
-    self._image = options.image;
+     self._image = options.image;
+    const processors = [ 'imagemagick', 'jimp' ];
     if (typeof (self._image) === 'string') {
-      self._image = require('./lib/image/' + self._image + '.js')();
+      if (processors.includes(self._image)) {
+        console.error('We now recommend the use of the sharp.js library for image processing. As fallback, if sharp is not installed we will attempt to use imagemagick instead.');
+      } else if (self._image != 'sharp') {
+        return callback('The sharp.js library is the default with imagemagick as the fallback. No other image processors are supported');
+      }
+    }
+
+    let fallback = false;
+    try {
+      self._image = require('./lib/image/sharp.js')();
```

Comment:
If `self._image` is already set to an object, not a string, we have no business touching it. That would prevent people from configuring custom image backends, which is allowed.

Reply:
Gotcha - misunderstood about passing it in as an object. Fixed in the new logic = self._image is only altered if it is a string, not touched if an object.